### PR TITLE
#336 moved emoji to variables

### PR DESCRIPTION
**Pull request by sakshamgurbhele**
# refactor static code variables

## Description
moved emoji to variables
## Issues:
Refs: #336 

## How has this been tested:
 - [x] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [x] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [x] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Review by sakshamgurbhele — COMMENTED** [review]
The issue is #336. sorry for that

File: src/tasks/litmus_setup.cr [hunk 1]
```diff
@@ -34,8 +34,11 @@ module LitmusManager
       puts "Checking experiment status  #{experimentStatus_cmd}" if check_verbose(args)
       experimentStatus = experimentStatus_response.to_s
       LOGGING.info "#{chaos_experiment_name} experiment status: "+experimentStatus
-      if (experimentStatus != "Waiting for Job Creation" && experimentStatus != "Running" && experimentStatus != "Completed")
-        resp = upsert_failed_task("pod-network-latency","✖️  FAILURE: #{chaos_experiment_name} chaos test failed 🗡️ 💀♻️")
+     
+      emoji_test_failed= "🗡️💀♻️"
+      if         
+        (experimentStatus != "Waiting for Job Creation" && experimentStatus != "Running" && experimentStatus != "Completed")
```
Comment (taylor) [inline src/tasks/litmus_setup.cr:40]:
```suggestion
      if (experimentStatus != "Waiting for Job Creation" && experimentStatus != "Running" && experimentStatus != "Completed")
```

File: src/tasks/litmus_setup.cr [hunk 2]
```diff
@@ -63,10 +66,11 @@ end
     puts "verdict: #{verdict_response.to_s}"  if check_verbose(args)  
     verdict = verdict_response.to_s 
 
+    emoji_test_failed= "🗡️💀♻️"
     if verdict == "Pass"
       true
-    else
-      puts "#{chaos_experiment_name} chaos test failed 🗡️💀♻️"
+    else
```
Comment (taylor) [inline src/tasks/litmus_setup.cr:72]:
```suggestion
    else
```

**Review by taylor — CHANGES_REQUESTED** [review]

File: src/tasks/litmus_setup.cr [hunk 3]
```diff
@@ -63,10 +65,10 @@ end
     puts "verdict: #{verdict_response.to_s}"  if check_verbose(args)  
     verdict = verdict_response.to_s 
 
+    emoji_test_failed= "🗡️💀♻️"
     if verdict == "Pass"
       true
-    else
-      puts "#{chaos_experiment_name} chaos test failed 🗡️💀♻️"
+    else puts "#{chaos_experiment_name} chaos test failed #{emoji_test_failed}"
```
Comment (taylor) [inline src/tasks/litmus_setup.cr:71]:
```suggestion
    else 
      puts "#{chaos_experiment_name} chaos test failed #{emoji_test_failed}"
```

**Review by taylor — CHANGES_REQUESTED** [review]
These else block should not have the puts on the same line.  The block is for the puts and false.

File: src/tasks/litmus_setup.cr (hunk 3 above)
Comment (taylor) [reply src/tasks/litmus_setup.cr:71]: @sakshamgurbhele You can click on "Commit suggestion"

**Comment by sakshamgurbhele** [comment]
> These else block should not have the puts on the same line. The block is for the puts and false.

Ohhkkkayyy

**Review by taylor — APPROVED** [review]

**Review by williscool — APPROVED** [review]
lgtm too.

will rebase and see if that fixes

**Comment by taylor** [comment]
current failing tests are caused because of lack of permissions for the CI system

**Pull request merged**